shopinvader_product_stock/models/shopinvader_backend.py

```diff
@@ -41,21 +43,39 @@ class ShopinvaderBackend(models.Model):
             "depending on the cron configuration"
         ),
     )
+    # NOTE: this field right now has no effect on qty sync.
+    # It's here to allow extending modules to define their own policies.
+    # See example in `shopinvader_product_stock_state`.
+    stock_level_config = fields.Selection(
+        selection="_selection_stock_level_config",
```

Comment:
@simahawk Why do you use a method to get the selectable values? Why not `selection=` and `selection_add=`?

Reply:
Because you get more control on it. Eg: you can sort items or remove items w/out hacks.